File: Find Closest Person/Find Closest Person.java
// Solution for Find Closest Person in JAVA

class Solution {
    public int findClosest(int x, int y, int z) {
        int person1diff = Math.abs(z - x);
        int person2diff = Math.abs(z - y);
        if (person1diff < person2diff) {
            return 1;
        } else if (person1diff > person2diff) {
            return 2;
        } 
        return 0;
    }
}

File: Find Closest Person/Find Closest Person.py
# Solution for Find Closest Person in PY

class Solution:
    def findClosest(self, x: int, y: int, z: int) -> int:
        person1diff, person2diff = abs(z - x), abs(z - y)
        if person1diff < person2diff:
            return 1
        elif person1diff > person2diff:
            return 2
        return 0
